internal/datastore/mysql/options.go

```diff
@@ -39,14 +39,15 @@ type Option func(*mysqlOptions)
 
 func generateConfig(options []Option) (mysqlOptions, error) {
 	computed := mysqlOptions{
-		gcWindow:            defaultGarbageCollectionWindow,
-		gcInterval:          defaultGarbageCollectionInterval,
-		gcMaxOperationTime:  defaultGarbageCollectionMaxOperationTime,
-		watchBufferLength:   defaultWatchBufferLength,
-		maxOpenConns:        defaultMaxOpenConns,
-		connMaxIdleTime:     defaultConnMaxIdleTime,
-		connMaxLifetime:     defaultConnMaxLifetime,
-		splitAtUsersetCount: defaultUsersetBatchSize,
+		gcWindow:              defaultGarbageCollectionWindow,
+		gcInterval:            defaultGarbageCollectionInterval,
+		gcMaxOperationTime:    defaultGarbageCollectionMaxOperationTime,
+		watchBufferLength:     defaultWatchBufferLength,
+		maxOpenConns:          defaultMaxOpenConns,
+		connMaxIdleTime:       defaultConnMaxIdleTime,
+		connMaxLifetime:       defaultConnMaxLifetime,
+		splitAtUsersetCount:   defaultUsersetBatchSize,
+		enablePrometheusStats: false,
```

Comment:
nit: do we want to pull a constant just like every other option?

Reply:
Changed